src/nanodbc.cpp

```diff
@@ -1851,12 +1896,9 @@ void statement::statement_impl::bind_strings(
             std::string narrow_rhs;
             narrow_rhs.reserve(s_rhs.size());
             convert(s_rhs, narrow_lhs);
-            if (std::strncmp(narrow_lhs.c_str(), narrow_rhs.c_str(), length))
```

Comment:
Removal of this `strncmp` leads to [MySQL build job 657.5](https://travis-ci.org/lexicalunit/nanodbc/jobs/170802339) failure on Linux 


```
/home/travis/build/lexicalunit/nanodbc/src/nanodbc.cpp:1900:66: 
    error: cannot convert ‘const char16_t*’ to ‘const char*’ for argument ‘1’ 
         to ‘int strncmp(const char*, const char*, size_t)’
    if (std::strncmp(s_lhs.c_str(), s_rhs.c_str(), length) != 0)
```